windows/src/desktop/kmshell/xml/strings.xml

```diff
@@ -10,7 +10,7 @@
   <!-- Context: System -->
   <!-- String Type: PlainText -->
   <!-- Introduced: 7.1.266.0 -->
-  <string name="S_LocaleAuthors" comment="Name(s) of people who created this translation - i.e. your name">Keyman</string>
+  <string name="S_LocaleAuthors" translatable="false" comment="Name(s) of people who created this translation - i.e. your name">Keyman</string>

```

Comment:
Maybe this is a legacy credits string, but should this still be translated?
Instead of Keyman maybe "your name"?

Reply:
good catch I will change it back